docs/spark-standalone.md

```diff
@@ -328,6 +328,17 @@ SPARK_WORKER_OPTS supports the following system properties:
   </td>
   <td>3.0.0</td>
 </tr>
+<tr>
+  <td><code>spark.shuffle.service.db.enabled</code></td>
+  <td>LEVELDB</td>
+  <td>
+    When <code>spark.shuffle.service.db.enabled</code> is true, user can use this to specify the kind of disk-based 
+    store used in shuffle state store. This supports `LEVELDB` and `ROCKSDB` now and `LEVELDB` as default value. 
+    This only affects standalone mode (yarn always has this behavior enabled). 
+    The original data store in `LevelDB/RocksDB` will not be automatically convert to another kind of storage now.
+  </td>
+  <td>3.4.0</td>
+</tr>
```

Comment:
Where is this being used ?
Also, `.enabled` typically indicates boolean

Reply:
Sorry, it's my fault. The name of this config should be `spark.shuffle.service.db.backend`.

Reply:
Should this be in spark-standalone ? applies to others as well, right ? (yarn for example ?)

Reply:
Yes, you are right ~ and `spark.shuffle.service.db.enabled` be used together

https://github.com/apache/spark/blob/a12d121159d0ab8293f70b819cb489cf6126224d/docs/spark-standalone.md?plain=1#L321-L328

Reply:
What I meant was, add this to 'External Shuffle service(server) side configuration options' section of configuration.md as well ?

Reply:
Got it.

Now `spark.shuffle.service.enabled` is in `configuration.md`, 
`spark.shuffle.service.db.enabled`and `spark.shuffle.service.db.backend` are in `spark-standalone.md`, 
should we add  both`spark.shuffle.service.db.enabled`and `spark.shuffle.service.db.backend` in  `configuration.md`?

Reply:
new diff is https://github.com/apache/spark/compare/db90b273fb941e9f32010562f91a644777605825..15b23255f955ef746fdd292b3fba67dff6c4cd4f

I copy both `spark.shuffle.service.db.enabledand` and  `spark.shuffle.service.db.backend` to `configuration.md `

Reply:
Yes, both - thanks